### ISSUE TITLE
GTM stories showing integration with Analytics not working

### ISSUE THREAD

**Issue by aaemnnosttv** [google/site-kit-wp#4666]
## Bug Description

Many stories for Tag Manager that show the integration with Analytics during module setup are not working properly (they just about all look the same).

E.g. Current Storybook
![image](https://user-images.githubusercontent.com/1621608/149820008-d67d2fd0-eb99-4c44-9062-851a0eea12e1.png)

compared to Storybook when these features were released (circa [1.18.0](https://github.com/google/site-kit-wp/releases/tag/1.18.0)) 3b90c45c8f2d3452935b53879a94a0bdf18beda3

![image](https://user-images.githubusercontent.com/1621608/149820151-632a535b-f489-43c2-b57a-98b4ea126e6c.png)

## Steps to reproduce

See https://google.github.io/site-kit-wp/storybook/main/?path=/story/tag-manager-module-setup--container-with-property-id-analytics-active-id-match

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The following Tag Manager stories that show Analytics integration should be fixed to correctly show the relevant notice since they are still relevant after ETS refactor. 
* The stories should display the relevant information/error message after choosing the account/property/container.  
* Since the messages are displayed only after the appropriate entity is selected, this should be done automatically in stories for correct demonstration. 
    * Tag Manager Module
        * Setup
            * Container with property ID, Analytics inactive
            * Container with property ID, Analytics active, ID match
            * Primary AMP
                * Container with property ID, Analytics inactive
                * Container with property ID, Analytics active, ID match
            * Secondary AMP
                * Singular property ID, Analytics inactive
                * Multiple property IDs, Analytics inactive
                * Multiple property IDs, Analytics active
                * Singular property ID, Analytics active, ID match
* The following story should be removed as they are no longer relevant after ETS refactor.
    * Tag Manager Module
        * Setup
            * Container with property ID, Analytics active, ID mismatch error
            * Primary AMP
                * Container with property ID, Analytics active, ID mismatch error
            * Secondary AMP
                * Singular property ID, Analytics active, ID mismatch error

## Implementation Brief

In `stories/modules-tagmanager-setup.stories.js`:

#### Fixing the stories by adding Mock Data:

* Tag Manager Module
  * Setup
      * `Container with property ID, Analytics inactive`
      * `Container with property ID, Analytics active, ID match`
      * For the above two stories, add the following mock data to the registry:
        * Get the `propertyID` from the `getLiveContainerAnalyticsPropertyID` selector.
        * Call the `createBuildAndReceivers.buildAndReceiveWebAndAMP` function by passing an object containing `webPropertyID` and `ampPropertyID` as the keys and the `propertyID` as the value.
        * Get the `webContainer` from the `getWebContainers` selector.
        * Set the container ID by dispatching the `setContainerID` action with the `webContainer.publicId` as the argument.
        * Set the internal container ID by dispatching the `setInternalContainerID` action with the `webContainer.containerId` as the argument.
    * Primary AMP
      * `Container with property ID, Analytics inactive`
      * `Container with property ID, Analytics active, ID match`
      * For the above two stories, add the following mock data to the registry:
        * Get the `propertyID` from the `getLiveContainerAnalyticsPropertyID` selector.
        * Call the `createBuildAndReceivers.buildAndReceiveWebAndAMP` function by passing an object containing `webPropertyID` and `ampPropertyID` as the keys and the `propertyID` as the value.
        * Get the `ampContainer` from the `getAMPContainers` selector.
        * Set the container ID by dispatching the `setAMPContainerID` action with the `ampContainer.publicId` as the argument.
        * Set the internal container ID by dispatching the `setInternalAMPContainerID` action with the `ampContainer.containerId` as the argument.
    * Secondary AMP
      * `Singular property ID, Analytics inactive`
      * `Multiple property IDs, Analytics inactive`
      * `Multiple property IDs, Analytics active`
      * `Singular property ID, Analytics active, ID match`
      * For the above four stories, add the following mock data to the registry:
        * Do the first two steps from the above stories.
        * Do both the `webContainer` and `ampContainer` steps from the above stories.

#### Remove the following stories:

* Tag Manager Module
  * Setup
    * Container with property ID, Analytics active, ID mismatch error
    * Primary AMP
      * Container with property ID, Analytics active, ID mismatch error
    * Secondary AMP
      * Singular property ID, Analytics active, ID mismatch error

### Test Coverage

* No new tests are to be added.

## QA Brief

* Verify the following Google Tag Manager setup stories are fixed:
  * Tag Manager Module
    * Setup
      * Container with property ID, Analytics inactive
      * Container with property ID, Analytics active, ID match
      * Primary AMP
        * Container with property ID, Analytics inactive
        * Container with property ID, Analytics active, ID match
      * Secondary AMP
        * Singular property ID, Analytics inactive
        * Multiple property IDs, Analytics inactive
        * Multiple property IDs, Analytics active
        * Singular property ID, Analytics active, ID match
* Verify the following Google Tag Manager setup stories are removed:
    * Tag Manager Module
      * Setup
        * Container with property ID, Analytics active, ID mismatch error
        * Primary AMP
          * Container with property ID, Analytics active, ID mismatch error
        * Secondary AMP
          * Singular property ID, Analytics active, ID mismatch error

## Changelog entry

* N/A

**Comment by aaemnnosttv** [comment]
@felixarntz many of these stories will need updating (or will be deleted) as part of #4448 so this probably shouldn't be done before that epic, but perhaps this issue will also no longer be relevant as the others are worked through.

**Comment by FlicHollis** [comment]
Hi @aaemnnosttv if you are not actively working on this, please could you un-assign yourself so someone else can pick this up? Thanks so much!

**Comment by aaemnnosttv** [comment]
@kuasha420 maybe you can look into this one? I'm wondering if this changed with the existing tag simplification and whether or not it's expected that these should continue to behave as before or not.

**Comment by bethanylang** [comment]
@kuasha420 Per Evan's comment above, reassigning this over to you.

**Comment by kuasha420** [comment]
@bethanylang Thank you, not sure whether I forgot or completely missed @aaemnnosttv's ping here. I'll have a look at this!

**Comment by tofumatt** [comment]
@kuasha420 I don't quite follow what notice should be shown for each of the stories mentioned… is there a previous commit we could reference to see how these should look, and then add to the AC that the stories should behave how they did at X commit ref (or plugin version number)?

**Comment by kuasha420** [comment]
@tofumatt thanks! I have added some more context in the AC regarding what needs to be done. Basically the scenarios have specific messages when the appropriate entities are selected, but currently nothing is selected so all lthat needs ooks the same. If we modify the stories to select the relevant entity for story, the story will behave correctly. Cheers.

**Comment by tofumatt** [comment]
@kuasha420 Thanks, the commit/release is especially helpful! 👍🏻 

ACs are good, moving to IB ✅

**Comment by mohitwp** [comment]
### QA Update ✅ 

- Verified on dev environment.
- Verified Tag Manager stories showing Analytics integration is now fixed to correctly show the relevant notice.
- Verified The stories display the relevant information/error message after choosing the account/property/container.

* Verified below listed stories are removed as they are no longer relevant after ETS refactor.
    * Tag Manager Module
        * Setup
            * Container with property ID, Analytics active, ID mismatch error
            * Primary AMP
                * Container with property ID, Analytics active, ID mismatch error
            * Secondary AMP
                * Singular property ID, Analytics active, ID mismatch error


<details>

Container with property ID, Analytics inactive

![image](https://github.com/google/site-kit-wp/assets/94359491/0bafbf79-e0a5-44c7-bcaa-30252c89a0d5)

Container with property ID, Analytics active, ID match

![image](https://github.com/google/site-kit-wp/assets/94359491/f2bf577d-d7c0-4b65-9208-70ad1fed03a2)


 * Primary AMP
                * Container with property ID, Analytics inactive
                
![image](https://github.com/google/site-kit-wp/assets/94359491/a5faf59b-851f-4585-a1c7-d2414084d591)

 * Primary AMP
                                  *Container with property ID, Analytics active, ID match

![image](https://github.com/google/site-kit-wp/assets/94359491/7455e5e6-0cfb-4c9c-92e0-52faec58f975)


*Secondary AMP
               *Singular property ID, Analytics inactive

![image](https://github.com/google/site-kit-wp/assets/94359491/23310814-21e6-4784-9b21-18af20e98b88)

   *Multiple property IDs, Analytics inactive

![image](https://github.com/google/site-kit-wp/assets/94359491/dec683bd-daf7-41fd-9350-e36ddfb5a0de)

*Multiple property IDs, Analytics active


![image](https://github.com/google/site-kit-wp/assets/94359491/dec585c2-e5c6-4bf3-990a-51f6ced1465d)

*Singular property ID, Analytics active, ID match

![image](https://github.com/google/site-kit-wp/assets/94359491/1c968319-4165-4dce-b713-5fa857fc311d)

</details>
